### ISSUE TITLE
[BUG] TSConfig Crawler configurations not working correctly

### ISSUE THREAD

**Issue by tomasnorre** [AOEpeople/crawler#242]
When adding the Crawler configurations via TSConfig, the crawler says no entries, when trying to "Update URLs". This means the crawler queue is empty. 

If the same configuration is added via Crawler Configuration Record it works like expected.

Versions: 
TYPO3 7.6 + Crawler: 5.2.0
TYPO3 8.7 + Crawler: 6.0.0

```
tx_crawler.crawlerCfg.paramSets.test >
tx_crawler.crawlerCfg.paramSets.test {
    cHash = 1
    procInstrFilter = tx_indexedsearch_reindex, tx_indexedsearch_crawler
    baseUrl = http://www.domain.tld/
}


tx_crawler.crawlerCfg.paramSets.grp1 < tx_crawler.crawlerCfg.paramSets.test
tx_crawler.crawlerCfg.paramSets.grp1{
    userGroups = 1
}
```

![screen shot 2017-12-08 at 11 58 53](https://user-images.githubusercontent.com/1212481/33763047-3929a532-dc0f-11e7-838d-b7f60c5b3bbe.png)

**Comment by tomasnorre** [comment]
Thanks for testing @KaffDaddy

**Comment by tomasnorre** [comment]
Is merged with PR #285
